### ISSUE TITLE
Switching between HTTPS and HTTP domains

### ISSUE THREAD

**Issue by jradxl** [straube/multiple-domain#41]
Hello and thanks for your plugin, which I have working with a slight addition fix.
I'm writing this in case it's a feature that could be added.
My website is available as https (let's encrypt) on my own domain into a Nginx reverse proxy.
The proxy relays the connection to a http 192.168.2.x server which runs WordPress.
I can connect to my WP instance from my laptop on my local LAN via http and of course connect via https from the internet (using my phone's 4G network).
It works except for layout issues via https.

To fix this https issue, I add the following code into wp-config.php
```
$_SERVER['HTTPS'] = 'off';
if ( (!empty( $_SERVER['HTTP_X_FORWARDED_HOST'])) || 
(!empty( $_SERVER['HTTP_X_FORWARDED_FOR'])) ) 
{ $_SERVER['HTTPS'] = 'on'; }
```
I notice that your configuration pages requires only the Domain to be entered, and not http://domain.com or https://domain.com
I was wondering if such a change would avoid the need for the additional code.
Unless there is a better way.
Thanks
John

**Comment by straube** [comment]
Hey John, I think this is a valid addition. Have you taken a look at this other issue https://github.com/straube/multiple-domain/issues/31? Do you think that protocol setting with Force HTTP, Force HTTPS, and Auto options would work in this case?

**Comment by straube** [comment]
@jradxl I released a new version (`0.10.0`) that includes a new option to set the protocol for a given domain. You can set it to `http`, `https` or `auto`. This last one will keep the protocol from the current domain, whatever it is `http` or `https`.

Let me know in case that update doesn't help you.
